gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -175,7 +175,9 @@ private static void configureErrorProneOptions(
                 "PreferImmutableStreamExCollections",
                 "UnusedVariable",
                 // See VarUsage: The var keyword results in illegible code in most cases and should not be used.
-                "Varifier");
+                "Varifier",
+                // Yoda style should not block baseline upgrades.
+                "YodaCondition");
```

Comment:
There are a few new 🌶️ checkers, I would bet on [AvoidObjectArrays](https://errorprone.info/bugpattern/AvoidObjectArrays), [Finalize](https://errorprone.info/bugpattern/Finalize), [YodaCondition](https://errorprone.info/bugpattern/YodaCondition) likely need some manual intervention/suppression upon rollout (though in general I think we want those checks enabled)

### 2.17:

[AvoidObjectArrays](https://errorprone.info/bugpattern/AvoidObjectArrays)
[Finalize](https://errorprone.info/bugpattern/Finalize)
[IgnoredPureGetter](https://errorprone.info/bugpattern/IgnoredPureGetter)
[ImpossibleNullComparison](https://errorprone.info/bugpattern/ProtoFieldNullComparison)
[MathAbsoluteNegative](https://errorprone.info/bugpattern/MathAbsoluteNegative)
[NewFileSystem](https://errorprone.info/bugpattern/NewFileSystem)
[StatementSwitchToExpressionSwitch](https://errorprone.info/bugpattern/StatementSwitchToExpressionSwitch)
[UnqualifiedYield](https://errorprone.info/bugpattern/UnqualifiedYield)

### 2.18:

[InjectOnBugCheckers](https://errorprone.info/bugpattern/InjectOnBugCheckers)
[LabelledBreakTarget](https://errorprone.info/bugpattern/LabelledBreakTarget)
[UnusedLabel](https://errorprone.info/bugpattern/UnusedLabel)
[YodaCondition](https://errorprone.info/bugpattern/YodaCondition)